docassemble/ALAutomatedTestingTests/al_set_up_testing.py

```diff
@@ -0,0 +1,48 @@
+from github import Github  # PyGithub
+import requests
+from nacl import encoding, public  # pynacl
+# TODO: Reduce to just one encryption library :/
+import codecs
+from base64 import b64encode
+import re
+import json
+from docassemble.base.util import log
+from docassemble.base.core import DAObject
```

Comment:
Are these all default python libraries? If not, it might be worth adding in a `requirements.txt` at some point that lists all the python libraries we use, so that developers can install them all in one go.

Reply:
Hmm, previously I had added the necessary dependencies on the Packages page. I think I forgot to do that this time around. Really good eye. The ones that are unique to our project, I think, are PyNacl and PyGithub. I'll have to ask around and see if others are non-standard.